### DISCUSSION TITLE
2020-12-31-year-review-2020/

### DISCUSSION THREAD

**Discussion by giscus** [sukso96100/blog.youngbin.xyz#30] [Blog Comments]
# 2020-12-31-year-review-2020/

https://blog.youngbin.xyz/2020-12-31-year-review-2020/

**Comment by ad-astra-per-ardua** [comment]
반갑습니다 영빈님 기억하실지는 모르겠는데 전역직전에 신삥으로 들어왔던 김성민입니다. 저도 전역하고 학부공부하고있는데 포폴 비슷한 사이트있으시다고 하셔서 한번 참고해보러왔습니다 여전히 대단하신분이시네요!
